src/orb.yml

```diff
@@ -344,6 +344,9 @@ jobs:
     executor: <<parameters.executor>>
     parallelism: <<parameters.parallelism>>
 
+    environment:
+      CYPRESS_CACHE_FOLDER: ~/.cache/Cypress
```

Comment:
using `~/` unfortunately works during install, but fails right now during `cypress verify` command, like in this build https://circleci.com/gh/cypress-io/cypress-example-kitchensink/3518

```
// install
> cypress@3.4.1 postinstall /Users/distiller/project/node_modules/cypress
> node index.js --exec install

Note: Overriding Cypress cache directory to: ~/.cache/Cypress

      Previous installs of Cypress may not be found.

Installing Cypress (version: 3.4.1)

[12:09:30]  Downloading Cypress     [started]
[12:09:33]  Downloading Cypress     [completed]
[12:09:33]  Unzipping Cypress       [started]
[12:09:53]  Unzipping Cypress       [completed]
[12:09:53]  Finishing Installation  [started]
[12:09:53]  Finishing Installation  [completed]

You can now open Cypress by running: node_modules/.bin/cypress open

https://on.cypress.io/installing-cypress
```

but then verify simply concatenates

```
#!/bin/bash --login -eo pipefail
npx cypress verify
The cypress npm package is installed, but the Cypress binary is missing.

We expected the binary to be installed here: /Users/distiller/project/~/.cache/Cypress/3.4.1/Cypress.app/Contents/MacOS/Cypress

Reasons it may be missing:
```

Reply:
I think the only thing that works is using absolute path, like `CYPRESS_CACHE_FOLDER: /Users/distiller/.cache/Cypress` which we should fix in Cypress (and resolve `~/...` paths)